wagtail/wagtailcore/tests/test_widgets.py

```diff
@@ -0,0 +1,40 @@
+from django.test import TestCase
+from wagtail.wagtailcore.fields import BaseTextAreaWidget, RichTextArea
+
+
+class TestBaseTextAreaWidget(TestCase):
+
+    def test_get_panel_implemented(self):
+        with self.assertRaises(NotImplementedError):
+            widget = BaseTextAreaWidget()
```

Comment:
This line should be not in `with` because `__init__` not a subject of this tests.